src/haddock/modules/refinement/openmm/defaults.yaml

```diff
@@ -149,7 +152,8 @@ solvent_boxsize_nm:
   max: 100
   title: Solvent boxsize in nm.
   short: Solvent boxsize in nm used to build the water solvation box around the molecule.
-  long: Solvent boxsize in nm used to build the water solvation box around the molecule. Value is used when the crystallographic cell size is removed in the pipeline.
+  long: Solvent boxsize in nm used to build the water solvation box around the molecule.
+  Value is used when the crystallographic cell size is removed in the pipeline.
```

Comment:
I think you need to indent for the `yam` to be valid: two spaces beyond the `long`.